geoportailv3/static/js/layerinfo/layerinfoservice.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @fileoverview Provides a layer info service. That service is a function used
+ * to retrieve the info (metadata) for a layer.
+ */
+
+goog.provide('app.ShowLayerinfo');
+
+goog.require('ngeo.CreatePopup');
+
+/**
+ * @typedef {function(ol.layer.Layer)}
+ */
+app.ShowLayerinfo;
+
+
+/**
+ * @param {angular.$http} $http Angular $http service
+ * @param {angular.$sce} $sce Angular $sce service
+ * @param {ngeo.CreatePopup} ngeoCreatePopup Ngeo popup factory service
+ * @ngInject
+ */
+app.showLayerinfoFactory = function($http, $sce, ngeoCreatePopup) {
+
+  /**
+   * @type {ngeo.Popup}
+   */
+  var popup = ngeoCreatePopup();
+
+  return (
+
+      /**
+       * @param {ol.layer.Layer} layer The layer
+       */
+      function(layer) {
+        popup.show();
+        var title = /** @type {string} */ (layer.get('label'));
+        popup.setTitle(title);
+        var content = /** @type {string} */
+            ($sce.trustAsHtml("<b>some</b> content"));
+        popup.setContent(content);
```

Comment:
It would make sense for now. But when we have a working service, I think will be nice to show a popup displaying a loading message while waiting for the request response.